projects/mci/inc/wavesculptor.h

```diff
@@ -16,61 +17,75 @@ typedef union WaveSculptorCanId {
   uint16_t raw;
 } WaveSculptorCanId;
 
-// Send to WaveSculptor
+// Send to WaveSculptor (Drive Commands)
 typedef enum {
-  WAVESCULPTOR_CMD_ID_DRIVE = 1,
-  WAVESCULPTOR_CMD_ID_POWER = 2,
-  WAVESCULPTOR_CMD_ID_RESET = 3,
+  WAVESCULPTOR_CMD_ID_DRIVE = 0x01,
+  WAVESCULPTOR_CMD_ID_POWER = 0x02,
+  WAVESCULPTOR_CMD_ID_RESET = 0x03,
 } WaveSculptorCmdId;
 
-// Driver Controls Base Addr + 1
+// Driver Controls Base Addr + 0x01
 typedef struct WaveSculptorDriveCmd {
-  float motor_velocity_ms;
+  float motor_velocity_rpm;
   float motor_current_percentage;
 } WaveSculptorDriveCmd;
 
-// Driver Controls Base Addr + 2
+// Driver Controls Base Addr + 0x02
 typedef struct WaveSculptorPowerCmd {
   uint32_t reserved;
   float bus_current_percentage;
 } WaveSculptorPowerCmd;
 
-// Driver Controls Base Addr + 3
+// Driver Controls Base Addr + 0x03
 typedef struct WaveSculptorResetCmd {
   uint64_t reserved;
 } WaveSculptorResetCmd;
 
-// Receive from WaveSculptor
+// Send to WaveSculptor (Configuration Commands)
+typedef enum {
+  WAVESCULPTOR_CONFIG_ID_ACTIVE_MOTOR = 0x12,
+} WaveSculptorConfigId;
+
+// Motor Controller Base Addr + 0x12
+typedef struct WaveSculptorActiveMotorChange {
+  // Must spell "ACTMOT" in ASCII (0x54 4F 4D 54 43 41)
+  uint16_t access_key[3];
+
+  // Desired active motor (0 to 9)
+  uint16_t active_motor;
+} WaveSculptorActiveMotorChange;
```

Comment:
Consider adding static_asserts for the size of these structs like you have below, unless these structs aren't sent verbatim to the wavesculptor?

Reply:
Thanks, agreed, will add/push asserts in a sec.

Re: sending to the WS -- we don't send those structs verbatim and instead use `motor_can.c` to pack drive commands, which are the only ones we send to the MC atm.  This is in `prv_handle_drive()` in `mci_output.c`.  We also use a `MotorCanDriveCommand` struct defined in `motor_can.h` instead of the `WaveSculptorDriveCmd` struct from `wavesculptor.h`, although I believe they're functionally the same.

I tested briefly and it seems like using a `wavesculptor.h` `WaveSculptorCanData` union, filling the `WaveSculptorDriveCmd` field with the velocity/current values and directly sending the union's raw data to the WS should work the same.  This is on x86 so it's little endian like the STM32F0 (I think), but there's probably hardware weirdness I'm not accounting for.

Should probably be a separate ticket, but I think it makes sense to deprecate the `motor_can.c` implementation so we're not carrying around two different drive command definitions, if you agree.